backend/portal/serializers.py

```diff
@@ -31,12 +31,14 @@ class Meta:
     def create(self, validated_data):
         club_code = validated_data["club_code"]
         # ensures user is part of club
+        '''TODO: Uncomment this, commented while testing
```

Comment:
Please leaves these changes unstaged before pushing. That way it won't show on the final code, but you'll still have it in your local setup